client/lib/feed-stream-store/paged-stream.js

```diff
@@ -249,16 +249,19 @@ export default class PagedStream {
 	}
 
 	dismissPostsFromSite( blogId ) {
```

Comment:
As far as I understood it: 
- the user is dismissing a recommendation _post_ not a blog
- the api dismisses the blog from recs as an implementation detail

Although I do agree that its weird to have the client do one thing while the backend has a different understanding.

Does the API know which post caused the site dismissal?